ql-qlf-k4n8-plugin/synth_quicklogic.cc

```diff
@@ -200,14 +209,16 @@ struct SynthQuickLogicPass : public ScriptPass {
             if (family == "qlf_k4n8") {
                 run("shregmap -minlen 8 -maxlen 8");
             }
-            run("techmap " + techMapArgs);
+            if (!noffmap) {
+                run("techmap " + techMapArgs);
+            }
             run("opt_expr -mux_undef");
             run("simplemap");
             run("opt_expr");
             run("opt_merge");
             run("opt_clean");
             run("opt");
-            run("dfflegalize -cell $_DFF_P_ x -cell $_DFF_P??_ x");
+            run("dfflegalize -cell $_DFF_P_ x -cell $_DFF_P??_ x -cell $_DFF_N_ x -cell _DFF_N??_");
```

Comment:
```suggestion
            run("dfflegalize -cell $_DFF_P_ x -cell $_DFF_P??_ x -cell $_DFF_N_ x -cell _DFF_N??_ x");
```

Reply:
That's correct provided that the initial state of a `_DFF_N??_` is unknown and the flip-flop does not support initialization. If that is not the case appropriate option from the `dfflegalize` documentation should be used here. Same for other flip-flop types.

Reply:
@kgugala @mkurc-ant looks like  _DFF_N??_ is not a valid type. where is the documentation for dfflegalize to check what types are supported?